shared/css/conversation.css

```diff
@@ -643,9 +709,39 @@ body[platform="win"] .share-service-dropdown.overflow > .dropdown-menu-item {
     left: auto;
     bottom: 0;
     width: 120px;
+  }
+
+  .media-wrapper:not(.showing-remote-streams) > .focus-stream > .local {
+    position: relative;
+    margin: 0;
+    bottom: auto;
+    width: 100%;
+    height: 100%;
+  }
+
+  .media-wrapper:not(.receiving-screen-share) > .focus-stream > .local > .avatar-volume {
+    height: 146px;
+    width: 146px;
+  }
+
+  .media-wrapper > .remote.focus-stream > .local .avatar-container {
+    border: 2px solid rgb(230, 230, 230);
+    border: 2px solid rgba(255, 255, 255, .9);
+    -webkit-background-clip: padding-box; /* for Safari */
+    background-clip: padding-box; /* for IE9+, Firefox 4+, Opera, Chrome */
     height: 120px;
   }
 
+  .media-wrapper.showing-remote-streams > .remote.focus-stream > .local .avatar-volume {
+    height: 106px;
+    width: 106px;
```

Comment:
I think it would be good to update the user story in the bug with the various situations & sizes the volume part should be. It'll be easier to review and ensure the css is correct that way.